### ISSUE TITLE
Text on Cardview search doesn't have sufficent contrast on the advanced filters 

### ISSUE THREAD

**Issue by joshuadkitenge** [ral-facilities/datagateway#951]
**Description**:
The text of the advanced filters is black on a black background 

![Screenshot 2021-11-26 090730](https://user-images.githubusercontent.com/83226114/143555651-7e049619-d87b-4ba3-a4de-746e012aeccc.png)

**Acceptance criteria**:
- [x] WCAG 2.1  1.4.3 contrast 1.4.6 contrast enhanced
